docs/tutorial.rst

```diff
@@ -29,113 +26,55 @@ The Yellowbrick library is a diagnostic visualization platform for machine learn
 About the Data
 --------------
 
-This tutorial uses a modified version of the mushroom dataset_ from
-the `UCI Machine Learning Repository <http://archive.ics.uci.edu/ml/>`__.
-Our objective is to predict if a mushroom is poisonous or edible based on
-its characteristics.
+This tutorial uses the mushrooms data from the Yellowbrick :doc:`api/datasets` module. Our objective is to predict if a mushroom is poisonous or edible based on its characteristics.
 
-.. _dataset: https://github.com/rebeccabilbro/rebeccabilbro.github.io/blob/master/data/agaricus-lepiota.txt
+.. NOTE:: The YB version of the mushrooms data differs from the mushroom dataset from the `UCI Machine Learning Repository <http://archive.ics.uci.edu/ml/>`__. The Yellowbrick version has been deliberately modified to make modeling a bit more of a challenge.
 
-The data include descriptions of hypothetical samples corresponding to
-23 species of gilled mushrooms in the Agaricus and Lepiota Family. Each
-species was identified as definitely edible, definitely poisonous, or of
-unknown edibility and not recommended (this latter class was combined
-with the poisonous one).
+The data include descriptions of hypothetical samples corresponding to 23 species of gilled mushrooms in the Agaricus and Lepiota Family. Each species was identified as definitely edible, definitely poisonous, or of unknown edibility and not recommended (this latter class was combined with the poisonous one).
 
-Our file, "agaricus-lepiota.txt," contains information for 3 nominally
-valued attributes and a target value from 8124 instances of mushrooms
-(4208 edible, 3916 poisonous).
+Our data contains information for 3 nominally valued attributes and a target value from 8124 instances of mushrooms (4208 edible, 3916 poisonous).
 
-Let's load the data with Pandas.
+Let's load the data:
 
 .. code:: python
 
-    import os
-    import pandas as pd
+    from yellowbrick.datasets import load_mushroom
 
-    names = [
-        'class',
-        'cap-shape',
-        'cap-surface',
-        'cap-color'
-    ]
-
-    mushrooms = os.path.join('data','agaricus-lepiota.txt')
-    dataset   = pd.read_csv(mushrooms)
-    dataset.columns = names
-    dataset.head()
-
-= =========  =========  ===========  =========
-. class      cap-shape  cap-surface  cap-color
-= =========  =========  ===========  =========
-0 edible     bell       smooth       white
-1 poisonous  convex     scaly        white
-2 edible     convex     smooth       gray
-3 edible     convex     scaly        yellow
-4 edible     bell       smooth       white
-= =========  =========  ===========  =========
+    X, y = load_mushroom()
+    print(X[:4]) # inspect the first five rows
```

Comment:
`X[:4`] only gives you the first 4 rows.  You can either use `X.head()` or `X[:5]` to retrieve first 5 rows

Reply:
@pdamodaran Please take another look at this

Reply:
I see that there were 5 rows printed.  It appears to look like 4 rows were printed because the counting started with 0.

Reply:
My mistake - should be 5 not 4